src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1027,7 +1027,7 @@ bool SequentialSfMReconstructionEngine::Resection(const size_t viewIndex)
     }
     if(!sfm::SfM_Localizer::RefinePose(
       optional_intrinsic.get(), pose,
-      resection_data, true, b_new_intrinsic))
+      resection_data, true, false))
```

Comment:
I think that delayed optimization will improve the results too. But I also agree that it will depends from dataset to dataset.
Since often you can use pre-calibrated camera, you will not have those problems.
But when you want to make some auto-calibration (from a guess) it's better to perform delayed refinement in order to maximize the observations per images in order to have an enhanced set of image observation.